README.md

```diff
@@ -142,7 +143,15 @@ FAQ
     iterate through the next few months, you'll see this repository focus more on integration and
     plugins, with more and more features becoming part of Kubernetes.
 
-2. What can I run on OpenShift?
+2. How does Atomic Enterprise relate to OpenShift?
+    
+    OpenShift Origin contains the full feature set of both Atomic Enterprise and OpenShift.  
```

Comment:
Build in order: AE, then openshift, then finish with the statement about origin containing all of both